services/token_lock.proto

```diff
@@ -34,9 +34,7 @@ import "basic_types.proto";
  */
  message TokenLockTransactionBody {
     AccountID account_id = 1;
-    TokenID token_id = 2; // token-definition-id or partition-definition-id
-    oneof amountOrSerialNumbers {
-        int64 amount = 3; // if token-definition is FUNGIBLE_COMMON
-        LockableSerialNumbers serial_numbers = 4; // if the token-definition is NON_FUNGIBLE_UNIQUE
-    }
+    TokenID token_id = 2;       // token-definition-id or partition-definition-id
+    int64 amount = 3;           // if token-definition is FUNGIBLE_COMMON
```

Comment:
Not sure why you changed oneof here. @Neeharika-Sompalli? A token can be either fungible or non_fungible.

Reply:
We decided to make this change consistent with the rest of the API. It is certainly not ideal. Using a `oneof` would be the better choice. However, in API design, consistency is usually considered more important.